cobspec/src/main/java/com/saralein/cobspec/controller/ParameterController.java

```diff
@@ -0,0 +1,47 @@
+package com.saralein.cobspec.controller;
+
+import com.saralein.server.controller.Controller;
+import com.saralein.server.parameters.ParameterDecoder;
+import com.saralein.server.parameters.ParameterParser;
+import com.saralein.server.request.Request;
+import com.saralein.server.response.Response;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+public class ParameterController implements Controller {
+    private final ParameterParser parser;
+    private final ParameterDecoder decoder;
+
+    public ParameterController(
+            ParameterParser parameterParser,
+            ParameterDecoder parameterDecoder) {
```

Comment:
Do you think that the controller should be parsing and decoding the parameters? It feels like something you would want to do during the request parsing phase.

Reply:
I was going to make it a middleware.  But I didn't think about just moving it into the request parser.  Will look into moving it there.